src/components/NftRewards/NftTierCard.tsx

```diff
@@ -13,6 +10,8 @@ import { ipfsToHttps } from 'utils/ipfs'
 import { NftPreview } from './NftPreview'
 import { QuantitySelector } from './QuantitySelector'
 
+const NFT_DISPLAY_HEIGHT = '141px'
```

Comment:
this feels overly specific. is there an existing tailwind class we can use for this? perhaps `h-32` or `h-36`?

Reply:
https://tailwindcss.com/docs/height

Reply:
The reason I was so specific is because that happens to be the width that results from the 3-col `Row` (of NFT cards) with a gutter of `24px`. I've reduced that gutter a bit so that the cards are now 144px wide instead of 141px, which means we can just use `h-36` to set the widths of the image/videos :)